addons/binding/org.openhab.binding.miele/ESH-INF/thing/dishwasher.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="miele"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Dishwasher -->
+	<thing-type id="dishwasher">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="xgw3000" />
+		</supported-bridge-type-refs>
+
+		<label>Dishwasher</label>
+		<description>This is a Miele@Home compatible dishwasher</description>
+
+		<channels>
+			<channel id="state" typeId="state" />
+			<channel id="program" typeId="program" />
+			<channel id="phase" typeId="phase" />
+			<channel id="start" typeId="start" />
+			<channel id="duration" typeId="duration" />
+			<channel id="elapsed" typeId="elapsed" />
+			<channel id="finish" typeId="finish" />
+			<channel id="door" typeId="door" />
+			<channel id="switch" typeId="switch" />
+		</channels>
+
+		<config-description>
+			<parameter name="uid" type="text">
+				<label>ID</label>
+				<description>The identifier identifies one certain appliance on the ZigBee network.</description>
+				<required>true</required>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+    <channel-type id="state" advanced="false">
```

Comment:
But this is not legal - you define two channel types with the SAME ID for this binding; this simply does not work.
Either you use a single one or you have to use different IDs like state1, state2, state3. But as they all look the same, I don't think that using a single one will cause any issues.